migrations/20191002105645-create-user.js

```diff
@@ -4,19 +4,40 @@ module.exports = {
     id: {
       allowNull: false,
       autoIncrement: true,
-      primaryKey: true,
       type: Sequelize.INTEGER,
     },
     firstName: {
       type: Sequelize.STRING,
     },
+    middleName: {
+      type: Sequelize.STRING,
+    },
     lastName: {
       type: Sequelize.STRING,
     },
+    role: {
+      type: Sequelize.STRING,
+      defaultValue: 'member',
+    },
+    membershipNo: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      allowNull: false,
+    },
     password: {
+      allowNull: false,
+      type: Sequelize.STRING,
+    },
+    dateOfBirth: {
+      type: Sequelize.STRING,
+    },
+    sex: {
+      type: Sequelize.STRING,
+    },
+    phoneNumber: {
       type: Sequelize.STRING,
     },
-    email: {
+    residentialAddress: {
```

Comment:
Why are we removing email? I think this will be the primary way of login.

Reply:
ok

Reply:
done